spec/models/spree/stock_item_spec.rb

```diff
@@ -3,7 +3,15 @@
 require 'spec_helper'
 
 RSpec.describe Spree::StockItem do
-  let(:stock_location) { create(:stock_location_with_items) }
+  let(:stock_location) { create(:stock_location) }
+
+  before do
+    product_1 = create(:product)
+    product_2 = create(:product)
+
+    stock_location.stock_items.where(:variant_id => product_1.master.id).first.adjust_count_on_hand(10)
+    stock_location.stock_items.where(:variant_id => product_2.master.id).first.adjust_count_on_hand(20)
```

Comment:
we can replace the hash rocket while we are at it :pray:

Reply:
well spotted :+1: